CMakeLists.txt

```
@@ -1014,7 +1014,7 @@ set(QT_GUI src/qt_gui/about_dialog.cpp
            src/qt_gui/cheats_patches.h
            src/qt_gui/compatibility_info.cpp
            src/qt_gui/compatibility_info.h
-	   src/qt_gui/control_settings.cpp	   
+	       src/qt_gui/control_settings.cpp	   
```

```suggestion
	   src/qt_gui/control_settings.cpp	   
```
clang-format didn't catch this but I did

---

This extra indent is still here